### ISSUE TITLE
Importing EIP by IP address appears to work, but doesn't really

### ISSUE THREAD

**Issue by Elethiomel** [hashicorp/terraform#8962]
I'd like to import an existing NAT gateway with its EIP intact. No matter what I do, the EIP is not imported and the nat gateway is recreated. This is important as the EIP is whitelisted at other sites.

### Terraform Version
Terraform v0.7.4-dev (968472a63e4542a2c5e2693c4489da4776dac996)

### Affected Resource(s)
aws_nat_gateway

### Terraform Configuration Files
```hcl
resource "aws_eip" "nat-eip-test" {
  vpc   = true
}

resource "aws_nat_gateway" "nat-gw" {
  subnet_id     = "subnet-12345678"
  allocation_id = "${aws_eip.nat-eip-test.id}"
}
```

### Expected Behavior
The NAT Gatway and EIP are imported. A terraform plan or apply does nothing as state matches the reality of AWS.

### Actual Behavior
Terraform destroys the imported NAT and creates a new NAT with a new EIP with the wrong address

### Steps to Reproduce
1) Terraform apply the above file
2) Remove the tfstate file
3) Import the EIP: terraform import aws_eip.nat-eip-test 52.1.1.1
4) Import the NAT Gatewat :  terraform import aws_nat_gateway.nat-gw nat-aaaaaaaaaa
4) terraform plan gives the below output

```hcl
+ aws_eip.nat-eip-test
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"

-/+ aws_nat_gateway.nat-gw
    allocation_id:        "eipalloc-abcdefg" => "${aws_eip.nat-eip-test.id}" (forces new resource)
    network_interface_id: "eni-abcabcab" => "<computed>"
    private_ip:           "10.0.0.1" => "<computed>"
    public_ip:            "52.1.1.1" => "<computed>"
    subnet_id:            "subnet-aaaaaa" => "subnet-aaaaaa"
```
5) terraform apply - This destroys the imported NAT, creates a new one and this new NAT has the wrong EIP

**Comment by catsby** [comment]
Thanks for writing in @Elethiomel – I think this is problem is two fold.

- most resources need to be imported by referencing their ID, and you've shown to be attempting to import the EIP by it's IP address. I tested this myself with a new EIP and confirmed that importing by it's `eipalloc-*` ID will behave correctly: `$ terraform import aws_eip.nat-eip-test  eipalloc-abcdefg` 
- It seems that importing an EIP by something like it's IP address appears to complete "successfully", but it uses the IP as the ID which doesn't make sense, and then doesn't get the right information in the statefile. This is a bug in EIP or EIP imports, which I patched in #8970
